Gemfile

```diff
@@ -25,7 +25,7 @@ gem 'pry-rails', '~> 0.3.4'
 gem 'rails_12factor', '~> 0.0.3', group: :production
 gem 'rack-cors', '~> 0.4.0', require: 'rack/cors'
 gem 'simple_token_authentication', '~> 1.6.0'
-
+gem 'paranoia'
```

Comment:
We have not discussed it. I agree with you that, most of the things that `paranoia` does, can be done manually, but this is just a training project, it's not so bad using `paranoia`